### DISCUSSION TITLE
[Feature] Reshare photos

### DISCUSSION THREAD

**Discussion by jensfriisnielsen** [immich-app/immich#5643] [Feature Request]
### The feature

I want to be able to add photos from albums shared to me to other shared albums. My use case is the following:

My family members each upload their photos of an event. We collaborate by adding all our personal event photos to a shared album. One of us does the work of curating and sharing a derived limited album with just the best photos. Others could do other derived albums.

Currently, my issue is that I cannot add photos that was shared to me to other albums. The UI lets me try it by selecting photo(s) -> Click + (Add) -> Add to shared album. However, shared photos are not added and the UI responds with "Info: Added 0 assets".

I don't know if this is a new feature or a bug. But I'd love to see the functionality! Personally I expect that if something has been shared with me I can have control of a copy.

Related requests:
- https://github.com/immich-app/immich/discussions/5013#discussion-5847421
- https://github.com/immich-app/immich/discussions/5359#discussion-5902064
- https://github.com/immich-app/immich/discussions/5457

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by aisbergde** [comment]
In General the possibilities on shared assets are very limited. I can add add only comments. I would like to mark as favorites. This should add them in any way to my assets (copy? link? ...). The goal is, that I can use these assets the same way, as they would be my own assets.
